bin/api-docs/update-api-docs.js

```diff
@@ -0,0 +1,233 @@
+/**
+ * External dependencies
+ */
+const { join, relative, resolve, sep, basename, dirname } = require( 'path' );
+const glob = require( 'fast-glob' );
+const execa = require( 'execa' );
+const { Transform } = require( 'stream' );
+const { readFile } = require( 'fs' ).promises;
+
+/**
+ * README file tokens, defined as a tuple of token identifier, source path.
+ *
+ * @typedef {[string,string]} WPReadmeFileTokens
+ */
+
+/**
+ * README file data, defined as a tuple of README file path, token details.
+ *
+ * @typedef {[string,WPReadmeFileTokens]} WPReadmeFileData
+ */
+
+/**
+ * Path to root project directory.
+ *
+ * @type {string}
+ */
+const ROOT_DIR = resolve( __dirname, '../..' );
+
+/**
+ * Path to packages directory.
+ *
+ * @type {string}
+ */
+const PACKAGES_DIR = resolve( ROOT_DIR, 'packages' );
+
+/**
+ * Path to data documentation directory.
+ *
+ * @type {string}
+ */
+const DATA_DOCS_DIR = resolve(
+	ROOT_DIR,
+	'docs/designers-developers/developers/data'
+);
+
+/**
+ * Pattern matching start token of a README file.
+ *
+ * @type {RegExp}
+ */
+const TOKEN_PATTERN = /<!-- START TOKEN\((.+?(?:\|(.+?))?)\) -->/g;
+
+/**
+ * Given an absolute file path, returns the package name.
+ *
+ * @param {string} file Absolute path.
+ *
+ * @return {string} Package name.
+ */
+function getFilePackage( file ) {
+	return file.startsWith( PACKAGES_DIR )
```

Comment:
I think this was most relevant prior to 37adca35027a1ba3bf7d16ecaff6b65c6772ba94 , where previously the path included in the token would try to resolve from the package root, rather than from where the documentation resided.

If it's not referenced elsewhere anymore, it may be safe to remove this condition.

Reply:
https://github.com/WordPress/gutenberg/pull/18840/commits/77dd061a875031d22bd87901c1461333e0a1c49e